test/connectivity/scripts/connectivityTest.sh

```diff
@@ -455,6 +471,12 @@ function usage() {
     exit 1;
 }
 
+is_build_canceled=$(is_cancel_build_requested $DEVOPS_ACCESS_TOKEN $DEVOPS_BUILDID)         
+if [ $is_build_canceled -eq 1 ]; then
+    print_highlighted_message "build is canceled."
+    exit 3
```

Comment:
use exit code 3 as it is the first custom exit code to use; for details, http://tldp.org/LDP/abs/html/exitcodes.html.